### PR TITLE
600 ViewProfile addition and ViewContact deprecation

### PR DESCRIPTION
**Pull request by pauldyson**
resolves: #600

My first go at this so feedback welcome!

ViewProfile added with various links from sidebars, the Contact and Organisation Context, etc.

ViewContact amended with a deprecation notice and removes from sidebar.

Updated spec.md to summarise all the standard intents and move ViewContact into a Deprecated section.

I haven't removed ViewContact from intents.ts or standard intents.js as it's not clear to me where they're used downstream

### CODE REVIEW

**Comment by kriswest** [comment]
> I haven't removed ViewContact from intents.ts or standard intents.js as it's not clear to me where they're used downstream

It shouldn't be removed from there until ViewContact is removed from the standard in a later version. Hence, this is fine.

File: docs/intents/ref/ViewContact.md [hunk 1]
```diff
@@ -4,6 +4,10 @@ sidebar_label: ViewContact
 title: ViewContact
 hide_title: true
 ---
+
+> **Note:** ViewContact has been deprecated in FDC 2.0 in favour of the more general [ViewProfile](ViewProfile)
+
+
 # `ViewContact`
```
Comment (kriswest) [inline docs/intents/ref/ViewContact.md:11]:
```suggestion
# `ViewContact`
> **Note:** ViewContact has been deprecated in FDC 2.0 in favour of the more general [ViewProfile](ViewProfile)

```

**Review by kriswest — APPROVED** [review]
LGTM, added a few minor suggestions

**Review by greyseer256 — APPROVED** [review]

**Review by mistryvinay — COMMENTED** [review]
Reviewing the use case detailed in #600 ,

"A salesperson is about to initiate a chat with a Buy Side fund manager on IM and just wants to check their details; job title, investment style, before they pick up the conversation. "

But ViewProfile is only returning name and emailAddress. Do we need to add the persons jobTitle?

File: docs/intents/ref/ViewContact.md [hunk 2]
```diff
@@ -5,6 +5,8 @@ title: ViewContact
 hide_title: true
 ---
 # `ViewContact`
+> **Note:** ViewContact has been deprecated in FDC 2.0 in favour of the more general [ViewProfile](ViewProfile)
```
Comment (kriswest) [inline docs/intents/ref/ViewContact.md:8]:
```suggestion
> **Note:** ViewContact has been deprecated in FDC3 2.0 in favour of the more general [ViewProfile](ViewProfile)
```

**Comment by mistryvinay** [comment]
> > But ViewProfile is only returning name and emailAddress. Do we need to add the persons jobTitle?
> 
> @mistryvinay As a 'View' intent (see discussion from the Context & Intents meeting when minutes are posted #627) its not actually expected to return data, but rather to start/use an app to display the data. The context object is actually an input that will be used to find the contact record to display. That said, further fields could be added to the context where relevant for retrieval - e.g. a IM username. However, that could be the subject of a separate PR (as the context can be expanded separately). If this has relevance to Symphony that could be an issue/PR you look to raise @mistryvinay.
> 
> We should expect to add a `GetProfile` intent in future that would actually return a context (either Contact or Organization), at which point it would indeed be good to expand those records to have more data fields in (than we need to search for/identify the contact)...

Thanks for the clarification @kriswest that makes sense, using this context object as an input for retrieving information. 👍

**Review by mistryvinay — APPROVED** [review]
Looks good to me.

**Pull request merged**